daml-lf/language/daml-lf.bzl

```diff
@@ -49,11 +50,14 @@ LF_VERSIONS = [
     "1.11",
     "1.12",
     "1.13",
+    "1.14",
     "dev",
 ]
 
 # All LF versions for which we have protobufs.
-PROTO_LF_VERSIONS = LF_VERSIONS
+# TODO https://github.com/digital-asset/daml/issues/8020
```

Comment:
are you intentionally leaving this out of this PR? Totally fine to do this separately just double checking because the PR is called "freeze lf 1.14".

Reply:
Yes.